p2p/switch.go

```diff
@@ -675,21 +644,27 @@ func (sw *Switch) acceptRoutine() {
 			break
 		}
 
-		nodeInfo, err := handshake(sw.nodeInfo, conn, sw.config.HandshakeTimeout)
+		stream, err := conn.OpenStream(HandshakeStreamID, nil)
```

Comment:
`HandshakeStreamID = 0` right now. I think we should use the maximum possible ID for it, to really avoid mess with channel/stream IDs used by reactors.